views/index.js

```diff
@@ -0,0 +1,16 @@
+const [filterByParty, filterByProvince] = document.querySelectorAll("select");
+
+filterByParty.addEventListener("input", filterResults);
+filterByProvince.addEventListener("input", filterResults);
+
+function filterResults(mp) {
+    document.startViewTransition(() => {
```

Comment:
This needs to be feature checked. Currently, if deployed, this won't work in Firefox (and some older versions of other browsers).